### PR TITLE
feat: Ren'py QA

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Basic QA to help catch translations of tokens

### CODE REVIEW

File: qa-checks/validate-renpy-strings.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { iterateThroughTests, createQaCheckFunction } = require("./utils");
+
+const tests = require("./__fixtures/validate-message-boxes-test-objects");
```
Comment (nhcarrigan) [inline qa-checks/validate-renpy-strings.test.js:3]: wow how did I miss this

**Review by ShaunSHamilton — APPROVED** [review]
Checks pass... 👍 🤷‍♂️ 

Only thing that irks me is the `validate-renpy-strings.js` file which, IMO, should export a function, and not be a standalone, but I am not sure how it is being called - _better not be `eval`ed_

**Comment by nhcarrigan** [comment]
image

**Review by ojeytonwilliams — APPROVED** [review]
This all LGTM.  Since the real test is "does this work in Crowdin?" can we get this in?

**Comment by nhcarrigan** [comment]
Whoops forgot this PR wasn't merged yet. 

The *actual* QA check has already been running on Crowdin. 😁

**Pull request merged**